django_tasks/backends/database/models.py

```diff
@@ -66,6 +66,9 @@ class DBTaskResult(GenericBase[P, T], models.Model):
         max_length=max(len(value) for value in ResultStatus.values),
     )
 
+    enqueued_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Question: Thoughts on exposing these in the admin? 

https://github.com/RealOrangeOne/django-tasks/blob/master/django_tasks/backends/database/admin.py#L11

They should definitely be available, but not sure if/which should show on the list page?

Reply:
I would like to see them in the list view. Should make debugging during development easier. I pushed an additional commit, which adds these to the list.

![Screenshot 2024-06-14 at 10 25 19](https://github.com/RealOrangeOne/django-tasks/assets/5329919/83e73c62-405d-4832-9d35-bd8db44341a8)